cmd/maintained-apps/validate/app_commander.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/pkg/file"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+type AppCommander struct {
```

Comment:
might be worth exploring embedding `fleet.MaintainedApp` here to DRY some of these fields up